micall/core/aln2counts.py

```diff
@@ -1137,10 +1154,18 @@ def write_nuc_report(self,
         report_nucleotides: typing.List[ReportNucleotide]
         for region, report_nucleotides in sorted(reports.items()):
             try:
-                gene = landmark_reader.get_gene(self.seed, region)
+                coordinate_name = landmark_reader.get_coordinates(seed)
+                gene = landmark_reader.get_gene(coordinate_name, region)
                 genome_start_pos = gene['start']
             except ValueError:
-                genome_start_pos = 1
+                try:
+                    coordinate_name = landmark_reader.get_coordinates(self.seed)
+                    gene = landmark_reader.get_gene(coordinate_name, region)
+                    genome_start_pos = gene['start']
+                except ValueError:
```

Comment:
Are there valid reasons to catch and warn on these exceptions, or should we just let them bubble up and fail the sample?
Same question on the other calls to landmark reader, below.

Reply:
I thought about why I included this detailed exception handling in the first place and in which cases no coordinates will be found, and I think for the combined and report nucleotides, this should not happen at all - so we can let these bubble up and fail the sample.  For the insertions, I left the exception handling in place, because it might come up with insertions relative to a contig that was only a partial match.